src/main/webapp/resources/js/pages/admin/components/user/CreateNewUser.jsx

```diff
@@ -0,0 +1,296 @@
+import { PagedTableContext } from "../../../../components/ant.design/PagedTable";
+import { AddNewButton } from "../../../../components/Buttons/AddNewButton";
+import { ScrollableModal } from "../../../../components/ant.design/ScrollableModal";
+import { validatePassword } from "../../../../utilities/validation-utilities";
+import { PasswordPolicyAlert } from "../../../../components/alerts/PasswordPolicyAlert";
+
+/**
+ * React component to display the create new user form.
+ * @returns {*}
+ * @constructor
+ */
+export default function CreateNewUser() {
+  const { updateTable } = React.useContext(PagedTableContext);
+  const { data: locales = [] } = useGetLocalesQuery();
+  const { data: systemRoles = [] } = useGetSystemRolesQuery();
+  const { data: emailConfigured = false } = useGetEmailConfiguredQuery();
+  const [createUser] = useCreateNewUserMutation();
+  const [form] = Form.useForm();
+  const [activationEmail, setActivationEmail] = React.useState(emailConfigured);
+  const [visible, setVisibility] = React.useState(false);
+  const [submitting, setSubmitting] = useState(false);
+  const [mailFailure, setMailFailure] = useState(false);
+  const usernameInput = React.useRef();
+  const isAdmin = window.TL._USER.systemRole === "ROLE_ADMIN";
+
+  useEffect(() => {
+    if (visible) {
+      usernameInput.current.focus();
+    }
+  }, [visible]);
+
+  const onFormFinish = (values) => {
+    setSubmitting(true);
+    createUser({ ...values })
+      .unwrap()
+      .then(() => {
+        notification.success({
+          message: i18n("CreateNewUser.notification.success"),
+          className: "t-user-page-notification-success",
+        });
+        form.resetFields();
+        setVisibility(false);
+        setMailFailure(false);
+        updateTable();
+      })
+      .catch((error) => {
+        notification.error({
+          message: i18n("CreateNewUser.notification.error"),
+        });
+        if (error.status === 400) {
+          const fields = Object.entries(error.data.errors).map(
+            ([field, error]) => ({
+              name: field,
+              errors: [error],
+            })
+          );
+          form.setFields(fields);
+        } else if (error.status === 409) {
+          form.setFieldsValue({ activate: false });
+          setActivationEmail(false);
+          setMailFailure(true);
+        }
+      })
+      .finally(() => setSubmitting(false));
+  };
+
+  return (
+    <>
+      <AddNewButton
+        className="t-add-user-btn"
+        onClick={() => setVisibility(true)}
+        text={i18n("CreateNewUser.button")}
+      />
+      <ScrollableModal
+        title={i18n("CreateNewUser.title")}
+        okText={i18n("CreateNewUser.form.button.submit")}
+        okButtonProps={{
+          className: "t-submit-btn",
+          htmlType: "submit",
+          loading: submitting,
+          onClick: form.submit,
+        }}
+        onCancel={() => setVisibility(false)}
```

Comment:
You close the modal you need to call form reset

Reply:
Please see 3067d5f280f82cbc740726a6a36e6644b4b77a8b.